superchain/init.go

```diff
@@ -59,6 +59,12 @@ func init() {
 			if err := yaml.Unmarshal(chainConfigData, &chainConfig); err != nil {
 				panic(fmt.Errorf("failed to decode chain config %s/%s: %w", s.Name(), c.Name(), err))
 			}
+
+			// For now we will only include chains declared as standard
+			if chainConfig.Type != "standard" {
+				continue
+			}
```

Comment:
Note that this means the chains marked "frontier" will be excluded from the Go module, and therefore from downstream modules including `validation`. 

We should think about how to handle this best. We could for example populate a different exported variable(s) with frontier chain information, allowing us to easily write validation checks which include all chain types.